examples/teams/team-riker/restrict-ingress-egress-riker.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
We need a PR for the SSP Patterns to highlight this feature.

Reply:
I get this PR'd on the SSP Patterns

Reply:
There is no PR atm. Since we don't have an issue for ssp-patterns PR, I assume it falls under the definition of done for this task.

Reply:
Feedback?

Reply:
Another issue will be opened here, with PR on SSP Patterns in the future.